dtkit/pom.xml

```diff
@@ -19,25 +19,6 @@
         </license>
     </licenses>
 
-   <developers>
```

Comment:
Could you revert this section? I know what jenkins suggest but I think it's right leave who contribute/create the plugin